docs/install.md

```diff
@@ -1,38 +1,64 @@
 # Installation
 
-We recommend **the easy way** if you are running your own server.
+There are many options for installing Sandstorm with various trade-offs. Choose the one that is most comfortable for you.
 
-## Installing the Easy Way
+Sandstorm requires Linux x86_64, with kernel version 3.13 or later.
 
-*Prerequisite:* Linux x86_64, with kernel version 3.13 or later.
+## Option 1: HTTPS-verified install
 
-To install on your own Linux machine, see
-[https://install.sandstorm.io/](https://install.sandstorm.io) or run:
+The easiest way to install Sandstorm is by running:
 
 ```bash
-    curl https://install.sandstorm.io | bash
+curl https://install.sandstorm.io | bash
 ```
 
-Note: If installing Sandstorm under LXC / Docker, you will need to choose the option to
-install as a non-root user. Unfortunately, this means the development tools will not
-work. This is due to the interaction between Sandstorm and Docker's use of Linux
-containerization features and missing features in the Linux kernel which we
-hope will be fixed eventually. For non-development purposes, Sandstorm should run just fine
-under Docker.
+## Option 2: Github-verified install
```

Comment:
Eh, I think I am OK with people taking install.sh from master.

The purpose of "github-verified install" is not to defend against our CDN but rather to defend against our server being hacked. Since github is essentially a public ledger it's much harder to push an attack script there. It seems like starting from github makes a lot of people more comfortable, in any case.

I think to anyone who cares it's clear enough that github is still HTTPS-verified so I don't think we need to complicate the heading.